app/src/main/java/org/oppia/android/app/profileprogress/ProfileProgressActivityPresenter.kt

```diff
@@ -52,7 +52,8 @@ class ProfileProgressActivityPresenter @Inject constructor(
   }
 
   fun openGalleryIntent() {
```

Comment:
Could you please update the coreesponding tests `testProfileProgressFragment_imageSelectAvatar_checkGalleryIntent` and `testProfileProgressFragment_imageSelectAvatar_configChange_checkGalleryIntent`

So that the pass?

Reply:
Successfully updated and passed two mentioned tests. @adhiamboperes you can check the updates.